tripleo_auto_abandon/auto_abandon.py

```diff
@@ -49,6 +49,17 @@ def load_config():
     CONF(['--config-file', 'auto-abandon.conf'])
 
 
+def _dry_run_msg(url, data):
+    return ("DRY RUN: POST %s DATA: %s" %(url, data))
+
+
+def purty_print(msg):
```

Comment:
Ultimately I think we'll want to replace this with actual Python logging, but for the moment I'm okay with adding it.